vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -602,6 +635,12 @@ def _prepare_prompt(
 
         # Internal state is managed here.
         slot_mapping = []
+
+        # TODO: we are deactivating all, because we
+        # only encode or prefill at time.
+        # self.input_batch.deactivate_all_requests()
```

Comment:
Shouldn't we do this instead here?

```suggestion
       self.input_batch.clear_requests()
```

Reply:
This method `deactivate_all_requests` does not exist anymore. It was something that tried, but did not work as expected. This approach was later replaced by the prefill_batch.